portfolio/src/main/webapp/script.js

```diff
@@ -63,7 +63,15 @@ function displayCaption(img) {
  */
 async function getMessageFromServlet(){
   const response = await fetch('/data');
-  const hello = await response.text();
-  document.getElementById('message-container').innerText = hello;
+  const messageData = await response.json();
+  console.log(messageData);
+  const messages = document.getElementById('message-container');
+  messages.innerHTML = '';
+  messageData.forEach(message => messages.appendChild(createParagraphElement(message)));
```

Comment:
Can you break up the contents of the forEach into a new line?

Reply:
Oops, I got the column limit for JS and Java confused again. Fixed!